rfc/0015-multi-file-jayvee/0015-multi-file-jayvee.md

```diff
@@ -0,0 +1,160 @@
+<!--
+SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+
+SPDX-License-Identifier: AGPL-3.0-only
+-->
+
+# RFC 0015: Multi-file Jayvee
+
+|             |                 |
+| ----------- | --------------- | --------------------------------------------------------------- |
+| Feature Tag | `multi-file`    |
+| Status      | `DRAFT`         | <!-- Possible values: DRAFT, DISCUSSION, ACCEPTED, REJECTED --> |
+| Responsible | `georg-schwarz` | <!-- TODO: assign yourself as main driver of this RFC -->       |
+
+<!--
+  Status Overview:
+  - DRAFT: The RFC is not ready for a review and currently under change. Feel free to already ask for feedback on the structure and contents at this stage.
+  - DISCUSSION: The RFC is open for discussion. Usually, we open a PR to trigger discussions.
+  - ACCEPTED: The RFC was accepted. Create issues to prepare implementation of the RFC.
+  - REJECTED: The RFC was rejected. If another revision emerges, switch to status DRAFT.
+-->
+
+## Summary
+
+This RFC introduces the possibility of distributing a Jayvee program over multiple files.
+This feature will foster reuse of valuetypes, blocks, and other elements.
+Inherent to this feature is a concept of how scoping and naming is handled for nested structures.
+We introduce two concepts in this RFC:
```

Comment:
Wording here is `we`, later on you use `I`. I would use neither, RFCs describe the state after they are accepted, not a suggestion. So this would be "Two concepts are introduced in this RFC:"